build

```
@@ -112,6 +120,8 @@ for PKG_PATH in $(catkin_topological_order --only-folders || colcon list --topol
   # https://github.com/ros-infrastructure/bloom/pull/643
   echo 11 > debian/compat
 
+  test -n "$DEB_ARCH" && set -- --arch="$OPTARG" "$@"
```

```suggestion
  test -n "$DEB_ARCH" && set -- --arch="$DEB_ARCH" "$@"
```
Sorry, my fault.

Also, can you move that bevoere the for loop in line 100?

---

This seems to work now. My CI is green.